hw/top_englishbreakfast/util/prepare_sw.py

```diff
@@ -182,6 +182,7 @@ def main():
         'build',
         '--features=-rv32_bitmanip',
         '--copt=-DOT_IS_ENGLISH_BREAKFAST_REDUCED_SUPPORT_FOR_INTERNAL_USE_ONLY_',
```

Comment:
And these are the --features and --copt you intended to pass for the risc V code generation. Along the line of @a-will's concern, is it safe to assume no other --copt flags will sneak in via some other kind of command? For example, the regular test rules will build both harness and device code, so if --copt was given in the `bazel test` invocation intended for the harness it may sneak through.

Reply:
The --copt and --features serve two purposes:
- The copts identify the platform as englishbreakfast and cause the preprocessor symbol `OT_IS_ENGLISH_BREAKFAST` to get defined (in `macros.h`).
- The features identify CPU code generation options for the specific rv32imc implementation in englishbreakfast.  Most notably, it disables the bitmanip and crc32 extensions.

Unfortunately, the copts currently _have_ to be passed through to host code generation to make sure the correct enumerations/defines are available for DIFs (e.g. rstmgr reasons) and for bindgen'ing the constants in to Rust opentitanlib/tool.

The feature flags matter less, but they also have no meaning to the host compiler - the bazel compiler configuration for the host doesn't have a feature called `rv32_bitmanip` and ignores it.

I don't particularly like this situation with copt leaking into other configurations, but until we change the overall hacky way englishbreakfast is handled in the codebase, we need it.  Eliminating all legacy usage of `opentitan_functest` in favor of the new rules will help us get to a configuration where englishbreakfast is a true separate top supported by exec_envs rather than by the current tree-dirtying operation.